ajax/libs/jquery-form-validator/package.json

```diff
@@ -33,8 +33,6 @@
       }
     ]
   },
-  "license": {
-    "type": "Dual licensed under the MIT or GPL Version 2 licenses",
-    "url": "http://www.gnu.org/licenses/gpl-2.0.html"
-  }
+  "license":"MIT",
```

Comment:
Let's add a space between `:` and `"`.
Like from A to B.
- A
```
"license":"MIT",
```
- B
```
"license": "MIT",
```
Thank you.